src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  text-align: center;
+  width: 200px;
+  border: 1px solid transparent;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  margin: 0 16px 4px;
+  text-align: left;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  color: var(--second-color);
+  margin-bottom: 16px;
+  margin-left: 16px;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__reviews {
+  margin-left: 17px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.card__price {
+  line-height: 18px;
+  font-size: 12px;
+  color: var(--second-color);
+}
+
+.card__price-item {
+  color: var(--main-color);
+  margin-left: 85px;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__stars-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 16px;
+  margin-bottom: 24px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 16px;
+}
+
+.card__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 16px;
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.card__link:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
```

Comment:
do not repeat yourself

you cand set transparent border, and just change color
remove `border-radius`, you already have it above

Reply:
Привіт, поясни будь ласка як я можу інакше записати бордер в ховері щоб не повторюватись. Я пробувала border-color але щось не працювало

Reply:
hi 9 point in checklist and if you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review